geth-poa/docker-compose.yml

```diff
@@ -250,11 +307,20 @@ services:
 networks:
   primev_net:
     external: true
+  l1_net:
```

Comment:
"networks are only created if they are required by at least one running service in your Compose file", so defining this won't affect the script in mev-commit that only brings up the settlement layer